uiowa/uiowa
# Removed slider block used in the v3 default content

Resolves #2331 

Believe this successfully removes the slider block from v3 default content. The last commit I made threw me off a bit (pretty sure that block content wasn't linked to other existing content? Then again I've been up all night so maybe I'm over analyzing things) but when I imported a fresh install of the default site everything worked as expected. 

# TO TEST
1. Do a fresh install of the default site
   1. blt drupal:install --site=default 
2. On the home page of the default site, make sure the slider and slides are removed and that there are no error messages (should go from ‘Welcome to your SiteNow site’ block to the ‘Getting started’ block)
3. Instead of the slider, there should now be three cards (‘Explore Features’, ‘Create Content’, and ‘Change Settings’) in a three column section under 'Getting Started'
